docOps/docs/specs/README.md

```diff
@@ -0,0 +1,1222 @@
+| checksum | ✔        | string  | Address checksum.                                                                                                     |
+
+
+#### Node
+
+| Property | Required | Type                                                          | Description                                                  |
+| -------- | -------- | ------------------------------------------------------------- | ------------------------------------------------------------ |
+| url      | ✔        | string                                                        | Node URL.                                                    |
+| pow      | ✔        | boolean                                                       | Determines if the node accepts proof of work.                |
+| username | ✘        | string                                                        | Node username. Only required if node requires authorisation. |
+| password | ✘        | string                                                        | Node password. Only required if node requires authorisation. |
+| network  | ✔        | <ul><li>`mainnet`</li><li>`devnet`</li><li>`comnet`</li></ul> | IOTA public network name.                                    |
+
+#### Timestamp
+
+| Property                    | Required | Type     | Description                                                                             |
+| --------------------------- | -------- | -------- | --------------------------------------------------------------------------------------- |
+| format(type: string):string | ✔        | function | Transaction timestamp in various formats.<br/> For example: MM-DD-YYYY, DD MM YYYY hh:mm:ss. |
+
+#### Transfer 
+
+Transfer object required for creating a transaction. It allows end-users to specify the transaction amount and recipient address.
+
+:::info
+Currently, it is not possible to send multiple payloads as part of the message. That is why the _tag_ property is omitted from this interface. You can find more details in this [GitHub pull request](https://github.com/iotaledger/protocol-rfcs/pull/18#discussion_r468432794).
+:::
+
+| Property       | Required | Type               | Description                    |
+| -------------- | -------- | ------------------ | ------------------------------ |
+| amount         | ✔        | number             | Transfer amount.               |
+| address        | ✔        | string             | Transfer address.              |
+| indexation_key | ✘        | Indexation Payload | (Optional) Indexation payload. |
+
+
+#### Value
+
+| Property                     | Required | Type     | Description                      |
+| ----------------------------- | -------- | -------- | -------------------------------- |
+| with_denomination():string    | ✔        | function | Transaction amount with unit.    |
+| without_denomination():number | ✔        | function | Transaction amount without unit. |
+
+
+#### Input
```

Comment:
```suggestion
#### Input

```
And on: keep an empty line after headers and in general between codeblocks/quotes/tables/paragraphs. Some engines render the md wrong if there are none.